src/process_utility.c

```diff
@@ -721,6 +696,15 @@ process_altertable_add_constraint(Hypertable *ht, const char *constraint_name)
 	foreach_chunk(ht, process_add_constraint_chunk, &hypertable_constraint_oid);
 }
 
+static void
+process_drop_constraint_chunk(Hypertable *ht, Oid chunk_relid, void *arg)
```

Comment:
`process_drop_chunk_constraint` better name?

Reply:
I think the unofficial naming strategy so far for processUtility stuff is process_<cmd>_<level>, where we basically append a finer grained suffix as we process the command and descend down the table hierarchy. So, we first do a high-level `process_drop_constraint()` to check for regular vs hypertable, then when we recurse to chunks and call `process_drop_constraint_chunk()`. I think this is mostly the scheme we are using for other commands too.

Reply:
ok